src/controllers/simplified-account/settings/team-members/remove-user/remove-user.controller.test.js

```diff
@@ -42,77 +32,87 @@ const viewOnlyUser = new User(userFixtures.validUserResponse(
     }
   }))
 
-const setupTest = (method, userToRemove, additionalReqProps = {}) => {
-  responseStub = sinon.spy()
-  renderErrorViewStub = sinon.spy()
-  findByExternalIdStub = sinon.stub().resolves(userToRemove)
-  deleteStub = sinon.stub().returns({ })
-
-  removeUserController = getController({
-    response: responseStub,
-    renderErrorView: renderErrorViewStub,
-    findByExternalId: findByExternalIdStub,
-    delete: deleteStub
+const mockResponse = sinon.spy()
+const mockRenderErrorView = sinon.spy()
+const mockFindByExternalIdGetsAdminUser = sinon.stub().resolves(adminUser)
+const mockFindByExternalIdGetsViewOnlyUser = sinon.stub().resolves(viewOnlyUser)
```

Comment:
you don't actually need to set up two separate stubs, you can just have one `mockFindUser` included in the `withStubs` in the builder, then set `mockFindUser.resolves(<admin user | view-only user>)` for each test, and then you don't need the `nextStubs()` calls

Reply:
(not that there's anything wrong with doing it this way, just a suggestion)

Reply:
thanks - makes so much sense that I've also applied this to the invite controller test